docker-run.sh

```diff
@@ -5,6 +5,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+# clean up exited containers
+docker rm $(docker ps -a -f name=automated-self-checkout -f status=exited -q)
```

Comment:
this should be in another docker run script `docker-run-dlstreamer.sh` to be more encapsulated